contents/docs/runbook/async-migrations/index.md

```diff
@@ -110,7 +110,7 @@ The table below lists out recommended PostHog app and chart versions to use for
 | 0001            | 1.33.0          | 16.1.0        |        |
 | 0002            | 1.33.0          | 16.1.0        |        |
 | 0003            | 1.33.0          | 16.1.0        |        |
-| 0004            | 1.36.1          | 26.0.0        | Run the async migration right after upgrading as there could be problems with ingestion otherwise | 
+| 0004            | 1.36.1          | 26.0.0        | This NOT the default PostHog version for v26 Chart version, see below. Run the async migration right after upgrading as there could be problems with ingestion otherwise | 
```

Comment:
```suggestion
| 0004            | 1.36.1          | 26.0.0        | This NOT the default PostHog version for v26 Chart version, see upgrade instructions below. Run the async migration right after upgrading as there could be problems with ingestion otherwise | 
```